Translate room classes (#30)
skyway/skyway-js-sdk-doc: docs/en/sfuroom.md

```js
room.once('log', logs => {
  for (const logStr of logs) {
    const { messageType, message, timestamp } = JSON.parse(logStr);
    // ...
  }
});
```

### Event: `'stream'`

Received MediaStream from remote peer in the room.
The Peer ID of stream origin can be obtained via `stream.peerId`.

| Name   | Type          | Description           |
| ------ | ------------- | --------------------- |
| stream | [MediaStream] | MediaStream instance. |

```js
room.on('stream', stream => {
  // ...
});
```

### Event: `'data'`

Received the data from remote peer in the room.

| Name | Type   | Description                                         |
| ---- | ------ | --------------------------------------------------- |
| data | object | [data object](#data-object) itself. |

#### data object

| Name | Type   | Description                     |
| ---- | ------ | ------------------------------- |
| src  | string | The Peer ID who sent this data. |
| data | *      | Sent data.                      |

```js
room.on('data', ({ src, data }) => {
  // ...
});
```

### Event: `'close'`

Room has closed.

```js
room.on('close', () => {
  // ...
});
```

### Event: `'removeStream'`

> MediaStream has removed from MediaConnection in this room.

Reviewer: i believe an SFURoom object does not have a MediaConnection. I would like to write this as follows, same as in Japanese.

```suggestion
MediaStream has removed from a media connection of this room.
```
